cfme/tests/openstack/infrastructure/test_provider.py

```diff
@@ -18,10 +19,15 @@ def test_api_port(provider):
     api_port = int(view_details.entities.summary('Properties').get_text_of('API Port'))
     assert api_port == port, 'Invalid API Port'
 
+
 def test_credentials_quads(provider):
     view = navigate_to(provider, 'All')
     prov_item = view.entities.get_entity(name=provider.name, surf_pages=True)
-    assert prov_item.data.get('creds') and 'checkmark' in prov_item.data['creds']
+    valid_message = 'Authentication credentials are valid'
+    if current_version() >= '5.10':
```

Comment:
Required:
you have `provider` object so no need to import `current_version`. you can just use `provider.appliance.version >="5.10"`